src/components/navigation/Hamburger.tsx

```diff
@@ -0,0 +1,46 @@
+
+export default function Hamburger({ isOpen }){
```

Comment:
To specify the type of the Hamburger Props we should create an interface like this:

```
interface HamburgerProps {
    isOpen: boolean;
}
```

And then in the component parameter we would write:

`({ isOpen }: HamburgerProps)`

Reply:
Fixed